api/eds.proto

```diff
@@ -142,8 +148,10 @@ message LocalityLbEndpoints {
 message EndpointLoadMetricStats {
```

Comment:
The contents of the above block need some attention. I have a Google internal design proposal for the backend reporting format, I will circulate this for wider design discussion when we have some idea of what makes sense for our needs. Meanwhile, please just remove the example backend load report at the top, and consider providing the aggregated report at the bottom in JSON format.